### ISSUE TITLE
Expose version or commit SHA of UI

### ISSUE THREAD

**Issue by tiffon** [jaegertracing/jaeger-ui#113]
Purpose is to be able to determine which version of the UI is currently deployed.

If version is used, it needs to not always be [`0.0.1`](https://github.com/jaegertracing/jaeger-ui/blob/4e02e451c7cd3f62154d4b5a6c8a0a0668281454/package.json#L3).

Thanks to @saminzadeh for suggesting this.

**Comment by yurishkuro** [comment]
+1. Is there an npm module that can extract the commit during the build?

**Comment by tiffon** [comment]
It would also be nice for this to show the version of the Jaeger services, not just the UI.

Related jaegertracing/jaeger#145

**Comment by tiffon** [comment]
@yurishkuro Re npm package, we might be able to use [git-repo-version](https://github.com/cibernox/git-repo-version) to generate (package-version, SHA, date) strings and [jest-changed-files](https://github.com/facebook/jest/tree/master/packages/jest-changed-files) to capture which files, if any, have been modified for the build.

**git-repo-version:** `1.5.0+a1b2c3d4e5 2016-10-24T18:26:53.000Z`

**jest-changed-files:**

```javascript
import {getChangedFilesForRoots} from 'jest-changed-files';

getChangedFilesForRoots(['/path/to/test'], {
  lastCommit: true,
  withAncestor: true,
}).then(files => {
  /*
  {
    repos: [],
    changedFiles: []
  }
  */
});
```

**Comment by yurishkuro** [comment]
during the build we can easily get the SHAs for both query service and the ui submodule using git commands directly. The UI build could read them from env vars.

**Comment by yurishkuro** [comment]
Resolved by #606
